reactor-aeron-tools/src/main/java/reactor/aeron/rsocket/netty/RSocketNettyServerTps.java

```diff
@@ -35,31 +35,48 @@
    * @param args program arguments.
    */
   public static void main(String... args) {
+    System.out.println(
+        "message size: "
+            + Configurations.MESSAGE_LENGTH
+            + ", number of messages: "
+            + Configurations.NUMBER_OF_MESSAGES
+            + ", address: "
+            + Configurations.MDC_ADDRESS
+            + ", port: "
+            + Configurations.MDC_PORT);
+
+    LoopResources loopResources = LoopResources.create("rsocket-netty");
 
     TcpServer tcpServer =
         TcpServer.create()
-            .runOn(LoopResources.create("server", 1, true))
+            .runOn(loopResources)
             .host(Configurations.MDC_ADDRESS)
-            .port(Configurations.MDC_PORT);
+            .port(Configurations.MDC_PORT)
+            .option(ChannelOption.TCP_NODELAY, true)
+            .option(ChannelOption.SO_KEEPALIVE, true)
+            .option(ChannelOption.SO_REUSEADDR, true)
+            .doOnConnection(System.out::println);
 
     RSocketFactory.receive()
         .frameDecoder(Frame::retain)
         .acceptor(
-            (setupPayload, rsocket) ->
-                Mono.just(
-                    new AbstractRSocket() {
-                      @Override
-                      public Flux<Payload> requestStream(Payload payload) {
-                        payload.release();
+            (setupPayload, rsocket) -> {
+              System.out.println(rsocket);
+              return Mono.just(
+                  new AbstractRSocket() {
+                    @Override
+                    public Flux<Payload> requestStream(Payload payload) {
+                      payload.release();
 
-                        Callable<Payload> payloadCallable =
-                            () -> ByteBufPayload.create(BUFFER.retainedSlice());
+                      int msgNum = (int) Configurations.NUMBER_OF_MESSAGES;
+                      System.out.println("streaming " + msgNum + " messages ...");
 
-                        return Mono.fromCallable(payloadCallable)
-                            .subscribeOn(Schedulers.parallel())
-                            .repeat(Configurations.NUMBER_OF_MESSAGES);
-                      }
-                    }))
+                      return Flux.range(1, msgNum)
+                          .map(i -> ByteBufPayload.create(BUFFER.retainedSlice()))
+                          .subscribeOn(Schedulers.parallel());
```

Comment:
FYI I am investigating why it doesn't work without `subscribeOn`, will post the result here soon-ish

Reply:
In the first place the issue is with memory leak ;  this server tps  thing doesn't last even 3 minutes

Reply:
there is no "leak", but the server for some reason does not send the payload to the client (yet drains the publisher)

/cc @OlegDokuka

Reply:
But, indeed, tps is much bigger with this code, but memory leak is killing it all.

Reply:
> there is no "leak"

it's not that leak which you are supposed to find with `ResourceLeakDetector`, but if go and look at jconsole you will see that heap grows and grows and grows

Reply:
maybe lets open an issue with this info ? so we can merge this one and re-run tests on aws?

Reply:
https://github.com/rsocket/rsocket-java/issues/514#issuecomment-467872520